README.md

```diff
@@ -4,22 +4,28 @@
 [![Gem Version](https://badge.fury.io/rb/rouge.svg)](https://rubygems.org/gems/rouge)
 [![YARD Docs](http://img.shields.io/badge/yard-docs-blue.svg)](https://rouge-ruby.github.io/docs/)
 
-[rouge]: http://rouge.jneen.net/
 
-[Rouge][] is a pure-ruby syntax highlighter.  It can highlight 100 different languages, and output HTML or ANSI 256-color text.  Its HTML output is compatible with stylesheets designed for [pygments][].
+[Rouge][] is a pure Ruby syntax highlighter. It can highlight over 100 different
+languages, and output HTML or ANSI 256-color text. Its HTML output is
+compatible with stylesheets designed for [Pygments][].
 
-If you'd like to help out with this project, assign yourself something from the [issues][] page, and send me a pull request (even if it's not done yet!).  Bonus points for feature branches.
+[Rouge]: http://rouge.jneen.net/ "Rouge"
 
-[issues]: https://github.com/rouge-ruby/rouge/issues "Help Out"
-[pygments]: http://pygments.org/ "Pygments"
+[Pygments]: http://pygments.org "Pygments"
 
 ## Usage
 
-First, take a look at the [pretty colors][].
+Rouge's most common uses are as a Ruby library, as part of Jekyll and as a
```

Comment:
```suggestion
Rouge's most common uses are as a Ruby library, as part of Jekyll, kramdown or Asciidoctor and as a
```

Reply:
This is describing Jekyll's most common usages. Asciidoctor is used by 3,305 repositories, Jekyll is 307,182. Kramdown doesn't list this information but I would suspect it's popularity is generally driven by Jekyll.

Reply:
The most common Rouge usage in Jekyll sites *could probably* be via the GFM-triple-backticks which in turn are parsed and rendered by `kramdown` or `commonmarker` whichever is the configured Markdown converter.

But I don't have solid stats on either. So :man_shrugging: